libs/designsystem/src/lib/components/modal/footer/modal-footer.component.scss

```
@@ -7,6 +7,10 @@ ion-footer {
   align-items: center;
   background-color: get-color('white');
   padding: size('s');
-  padding-bottom: calc(#{size('s')} + var(--keyboard-offset, 0px) * var(--snap-to-keyboard, 0));
+  padding-bottom: size('s');
   transition: padding-bottom $soft-keyboard-transition;
 }
+
+:host(.snap-to-keyboard) ion-footer {
+  padding-bottom: calc(#{size('s')} + var(--keyboard-offset, 0px));
+}
```

Move transition under `.snap-to-keyboard` selector:
```suggestion
}

:host(.snap-to-keyboard) ion-footer {
  padding-bottom: calc(#{size('s')} + var(--keyboard-offset, 0px));
  transition: padding-bottom $soft-keyboard-transition;
}
```